### DISCUSSION TITLE
Theme customization help

### DISCUSSION THREAD

**Discussion by LightningRpper** [ohmyzsh/ohmyzsh#10588] [Help] [answered]
I'm trying to change the ZSH theme to my user name "jstevenson" instead of "robbyrussell" so I can build a website my own way, but every time I change the ZSH_THEME name I get an error saying` /home/jstevenson/.oh-my-zsh/.oh-my-zsh.sh:source:176: no such file or directory: /home/jstevenson/.oh-my-zsh/themes/jstevenson.zsh-theme.`
What should I do to fix this?

**Comment by mcornella** [comment, marked as answer]
You need to copy the robbyrussell theme (at `themes/robbyrussell.zsh-theme`) and change its name to yours. See [the guide here](https://github.com/ohmyzsh/ohmyzsh/wiki/Customization#overriding-and-adding-themes).
